### PR TITLE
Made nano the default editor in all 3 JupyterLab images

### PR DESCRIPTION
**Pull request by xoen**
Made `nano` the default editor as per ticket: https://trello.com/c/xmE6eamr

The default was `jed` so we're moving to a more common and battled-tested command line editor here.

### CODE REVIEW

File: datascience-notebook/tests/controls/nano_spec.rb [hunk 1]
```diff
@@ -0,0 +1,19 @@
+title 'nano is available and default editor'
+
+control 'nano available and default editor' do
+  impact 'low'
```
Comment (damacus) [inline datascience-notebook/tests/controls/nano_spec.rb:4]: This should be a numeric value from 0-1 .

e.g. 0.1, or 1

Comment (damacus) [inline datascience-notebook/tests/controls/nano_spec.rb:4]: Huh, TIL, impact levels can also be strings! I've always used numerical values.

Comment (xoen) [reply datascience-notebook/tests/controls/nano_spec.rb:4]: Love how I accidentally discovered this on day 0 😬

**Comment by xoen** [comment]
Thanks for the review @damacus ❤️

**Pull request merged**